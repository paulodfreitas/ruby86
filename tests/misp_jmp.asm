.module entry_code
.pseg
.org 0
        irmovl $3000, %eax
        subl %eax, %eax
        jne if
else:
        irmovl $1, %ecx
        jmp end
if:
        irmovl $0, %ecx
end:
        halt
.end
