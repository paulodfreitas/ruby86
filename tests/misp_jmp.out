@ 0000
30
80
00
30
00
00
61
00
74
18
00
00
00
30
81
01
00
00
00
70
1e
00
00
00
30
81
00
00
00
00
10
@ 001f
